specification/quantum/resource-manager/Microsoft.Quantum/preview/2023-11-13-preview/quantum.json

```diff
@@ -0,0 +1,1117 @@
+          "description": "Link to the next set of results. Not empty if Value contains incomplete list of Workspaces."
+        }
+      },
+      "description": "The response of a list Workspaces operation."
+    },
+    "OfferingsListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ProviderDescription"
+          },
+          "description": "Result of a list Providers operation."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains incomplete list of Providers."
+        }
+      },
+      "description": "The response of a list Providers operation."
+    },
+    "QuantumWorkspace": {
+      "type": "object",
+      "description": "The resource proxy definition object for quantum workspace.",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Gets or sets the properties. Define quantum workspace's specific properties.",
+          "$ref": "#/definitions/WorkspaceResourceProperties"
+        },
+        "identity": {
+          "type": "object",
+          "description": "Managed Identity information.",
+          "properties": {
+            "principalId": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The principal ID of resource identity."
+            },
+            "tenantId": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The tenant ID of resource."
+            },
+            "type": {
+              "type": "string",
+              "description": "The identity type.",
+              "enum": [
+                "SystemAssigned",
+                "None"
+              ],
+              "x-ms-enum": {
+                "name": "ResourceIdentityType",
+                "modelAsString": true
+              }
+            }
+          }
+        },
+        "systemData": {
+          "description": "System metadata",
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/systemData",
+          "readOnly": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "ApiKey": {
+      "description": "Result of Api key.",
+      "type": "object",
+      "properties": {
+        "createTime": {
+          "type": "string"
+        },
+        "key": {
+          "x-ms-secret": true,
+          "readOnly": true,
+          "type": "string",
+          "description": "The Api key."
+        }
+      }
+    },
+    "ListKeysResult": {
+      "description": "Result of list Api keys and connection strings.",
+      "type": "object",
+      "properties": {
+        "apiKeyEnabled": {
+          "type": "boolean",
```

Comment:
Please check if it should be an enum instead of boolean, as an enum is extensible (with x-ms-enum) and gives ability to provide more than two possible values in future without needing any code changes and breaking changes.

Reply:
Yes, it should be boolean. The two possible values are only true and false standing for enabled and disabled.